package.json

```diff
@@ -17,50 +21,41 @@
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.26.1",
     "eslint-plugin-react-hooks": "^4.1.2",
-    "husky": "^7.0.4",
-    "lerna": "^4.0.0",
-    "lint-staged": "^10.4.0",
     "prettier": "^2.1.2",
     "scramjet": "^4.35.1",
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^8.0.2",
     "stylelint-config-sass-guidelines": "^8.0.0",
     "stylelint-prettier": "^1.1.2",
-    "typescript": "^4.4.3"
+    "typescript": "^4.4.3",
+    "rimraf": "^4.1.2"
+  },
+  "engines": {
+    "node": "16.14.2"
   },
   "scripts": {
     "prepare": "husky install",
-    "preinstall": "npm run check:node",
-    "prebootstrap:install": "npm run check:node",
-    "bootstrap": "npm run bootstrap:install -- --ci",
-    "bootstrap:prod": "npm run bootstrap:install -- --ci -- --only=prod",
-    "bootstrap:install": "lerna bootstrap --force-local",
-    "clean": "lerna clean -y",
-    "check:node": "check-node-version --node 16 --npm 8",
-    "fix": "npm run fix:js && npm run fix:style",
+    "clean": "pnpm -r --parallel exec rimraf node_modules && rimraf node_modules",
```

Comment:
A simpler command would be `rimraf **/node_modules`? Don't need `pnpm` as `rimraf` is a Node binary.

Reply:
As discussed in the office it looks like rimraf doesn't support glob matching & we meed to delete the root `node_modules` folder last in order to be able to run `rimraf`